inginious-webapp

```diff
@@ -89,4 +90,15 @@ if __name__ == "__main__":
 
     # Launch the app
     if __name__ == "__main__":
+
+        reverse_proxy_config = config.get('reverse-proxy-config',{})
```

Comment:
Looks good, but I'd rather put all this before the if statement, so that this is also applied if the app is launched through a wsgi server such as uwsgi for instance.

Reply:
Ok, Got it